meetups/hyderabad/02-jingle-bells/README.md

```diff
@@ -40,5 +40,6 @@ organizers:
 ---
 
 <EventPage />
-## Google Photos Album 
-[https://photos.app.goo.gl/BU6QXzECrQBTG2dB9](https://photos.app.goo.gl/BU6QXzECrQBTG2dB9)
+## Google Photos Album
+https://photos.app.goo.gl/BU6QXzECrQBTG2dB9
```

Comment:
```suggestion
https://photos.app.goo.gl/dKY5yXg9URmKt1Ko6
```

Reply:
@znck please commit this suggestion. Also, don't have push permission. probably something wrong with the codeowners setup.